harvester/lib/cf_handler.py

```diff
@@ -71,9 +72,16 @@ def get_all_app_tasks(self, app_guuid=None):
         self.setup()
         # get tasks returns a single "page" and we need to manually wrap it into a
         # pagination object
-        return list(
-            self.client.v3.apps._pagination(self.client.v3.apps.get(app_guuid, "tasks"))
-        )
+        try:
+            result_iter = self.client.v3.apps._pagination(
+                self.client.v3.apps.get(app_guuid, "tasks")
+            )
+        except InvalidStatusCode:
+            logger.warning(
+                "Failed to get app tasks from CF, task information is not accurate"
+            )
+            return []
```

Comment:
So if we return an empty array here, won't this then spin up a full set of jobs (say 3)? And on next startup, if the Task list is still down, it will spin up 3 more?
Just to be clear, we have https://github.com/GSA/datagov-harvester/blob/main/harvester/lib/load_manager.py#L91C38-L91C59, which gets from num_running_app_tasks, which calls get_running_app_tasks, which calls this function. We need a way to propagate that up in a way that allows us to recognize we shouldn't spin up new tasks, I would think.

Reply:
Sigh, yes that could happen. I re-plumbed everything up from `get_all_app_tasks` using `None` as a sentinel value for when the API isn't available. Please re-review.